extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -226,20 +229,32 @@ void ConsumeWindowsEventLog::onSchedule(const std::shared_ptr<core::ProcessConte
     }
   }
 
+  std::string mode;
+  context->getProperty(OutputFormat.getName(), mode);
+
+  writeXML_ = (mode == Both || mode == XML);
+
+  writePlainText_ = (mode == Both || mode == Plaintext);
+
+  if (writeXML_) {
+    char systemDir[MAX_PATH];
+    if (GetSystemDirectory(systemDir, sizeof(systemDir))) {
+      hMsobjsDll_ = LoadLibrary((systemDir + std::string("\\msobjs.dll")).c_str());
+      if (!hMsobjsDll_) {
```

Comment:
It's nice to have error handling here, although we should make sure not to call "FormatMessage" later in case this dll open failed.

Reply:
It is checked in the beginning of substituteXMLPercentageItems.